### ISSUE TITLE
Unity Error: Multiple precompiled assemblies with the same name Newtonsoft.Json.dll

### ISSUE THREAD

**Issue by craigcook04** [MicrosoftDocs/mslearn-mr-adt-in-unity#8]
Thank you Adam and Brent for this session at Build, great presentation and really interesting topic!

I managed to follow along with the tutorial up to the point of creating my Unity project from the folder 'Unity-Project' cloned from this repo. I get the following error when loading my Unity project in version 2020.3.11f1:

"Multiple precompiled assemblies with the same name Newtonsoft.Json.dll included or the current platform. Only one assembly with the same name is allowed per platform."

It seems as though this file is located in 'Library\PackageCache\com.unity.nuget.newtonsoft-json@2.0.0\Runtime' and 'Assets\Plugins'.

Any assistance would be much appreciated!

**Comment by arfontai** [comment]
Hi craigcook04,

I experienced the same issue.
Just remove manually one of the two references while you are in Unity Safe Mode. It solved the issue on my side.

**Comment by craigcook04** [comment]
Thank you @arfontai for the quick response.

I fixed the error using your suggestion, and now a new error appears:

System.IO.DirectoryNotFoundException: Could not find a part of the path 'mslearn-mr-adt-in-unity\Unity-Project\Library\PackageCache\com.microsoft.maps.unity@0.10.1\Plugins\CompressionNative\iOS\MicrosoftGeospatialDecompression.framework\_CodeSignature'

I manually navigated through my file structure and found that the folder does exist in the location as it says above.

My project no longer starts up in safe mode by default, and this error still persists when I reload my project.

Thank you!
